prov/efa/Makefile.include

```diff
@@ -73,9 +73,26 @@ _efa_headers = \
 	prov/efa/src/rxr/rxr_atomic.h \
 	prov/efa/src/rxr/rdm_proto_v4.h
 
+
+if ENABLE_EFA_UNIT_TEST
```

Comment:
When I configure with for unit testing, I still need to manually add the cmocka library path to my LD_LIBRARY_PATH for `make check` to work.  The output of `make check` is dense and hard to parse visually, and I think the tester needs to `cat prov/efa/test/efa_unit_test.log` to see what went wrong.

Reply:
Fixing this with --cmocka <path>, and adding that to rpath.